tests/app/dao/test_fact_notification_status_dao.py

```diff
@@ -674,10 +674,12 @@ def test_fetch_delivered_notification_stats_by_month_empty():
     assert fetch_delivered_notification_stats_by_month() == []
 
 
-@freeze_time('2020-11-02 14:00')
 def test_fetch_notification_stats_for_trial_services(sample_service):
-    trial_service = create_service(service_name='restricted', restricted=True)
-    trial_service_2 = create_service(service_name='restricted_2', restricted=True)
+    with freeze_time('2020-11-02 14:00'):
+        trial_service = create_service(service_name='restricted', restricted=True)
+
+    with freeze_time('2020-11-01 14:00'):
+        trial_service_2 = create_service(service_name='restricted_2', restricted=True)
```

Comment:
Unrelated to the feature, but fixes a flaky test, introduced in https://github.com/cds-snc/notification-api/pull/1173/commits/4c72b26edf4286e66307bc8d1e3ff3e5f6514133